components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -86,6 +82,8 @@ object Glean {
 
         initializeCoreMetrics(applicationContext)
 
+        Baseline.os.set("Android")
```

Comment:
Can we move this inside `initializeCoreMetrics`, right before the end?
Moreover, can we initialize (or comment out or remove) the other metrics defined in the `metrics.yaml` file? I'm fine with commenting them out and taking their initialization as part of a separate bug. Initializing these metrics could be part of [bug 1497894](https://bugzilla.mozilla.org/show_bug.cgi?id=1497894)